lib/datadog/ci/transport/api/builder.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+require_relative "ci_intake"
+require_relative "evp_proxy"
+
+module Datadog
+  module CI
+    module Transport
+      module Api
+        module Builder
+          def self.build_ci_test_cycle_api(settings)
+            dd_site = settings.site || "datadoghq.com"
```

Comment:
This could (or should?) come from constants (like the hot prefix below?)

Reply:
this is true, will define a DEFAULT_DD_SITE constant